app/javascript/components/applications/application_form/PrimaryApplicantSection.js

```diff
@@ -102,22 +68,20 @@ const PrimaryApplicantSection = ({formApi, editValues }) => {
             </div>
             <div className="row">
               <div className="small-4 columns">
-                <div className={classNames('form-group', errorClassName(formApi,'DOB'))}>
-                  <label className='form-label'>Date of Birth</label>
-                  <Text
-                    field="DOB"
-                    type="date"
-                    pattern="[0-9]{4}-[0-9]{2}-[0-9]{2}"
-                    placeholder="YYYY-MM-DD"
-                    className={classNames(errorClassName(formApi, 'DOB'))} />
-                  <FormError formApi={formApi} field='DOB'/>
-                </div>
+                <Field formApi={formApi} field='DOB' label='DOB' blockNote='- YYYY-MM-DD (required)' >
```

Comment:
this block note is actually incorrect for the way the date is input in chrome with the HTML5 date field:

![image](https://user-images.githubusercontent.com/11825994/44112809-ff780bec-9fba-11e8-9f16-f679f4ceb1ba.png)